keyboards/3.0/fr-t-k0-azerty.xml

```diff
@@ -153,54 +153,63 @@
 		</layer>
 	</layers>
 
+	<variables>
+		<string id="grave" value="`" />
+		<string id="caret" value="^" />
+		<string id="umlaut" value="¨" />
+		<string id="tilde" value="~" />
+		<set id="visible_accent" value="${grave} ${caret} ${umlaut} ${tilde}" />
```

Comment:
visible_accent may be better called spacing_accent (other accents aren't invisible, just combining)